backend/src/test/app.test.ts

```diff
@@ -16,201 +16,308 @@ afterEach(() => {
     jest.clearAllMocks();
 });
 
-describe("Post Election", () => {
+// describe("Post Election", () => {
+// 
+//     describe("Election data provided", () => {
+//         test("responds with 200 status", async () => {
+//             const response = await request(app)
+//                 .post('/API/Elections')
+//                 .set('Cookie', ['id_token=' + testInputs.user1token])
+//                 .set('Accept', 'application/json')
+//                 .send({ Election: testInputs.Election1, VoterIDList: [] })
+// 
+//             expect(response.statusCode).toBe(200)
+//         })
+//         test("Get responds with 200 status", async () => {
+//             const response = await request(app)
+//                 .get('/API/Election/0')
+//                 .set('Cookie', ['id_token=' + testInputs.user1token])
+//                 .set('Accept', 'application/json')
+//             // console.log(response)
+//             expect(response.statusCode).toBe(200)
+//         })
+//     })
+//     describe("Election not provided/incorrect format", () => {
+//         test("responds with 400 status", async () => {
+//             const response = await request(app)
+//                 .post('/API/Elections')
+//                 .set('Cookie', ['id_token=' + testInputs.user1token])
+//                 .set('Accept', 'application/json')
+//                 .send({ VoterIDList: [] })
+// 
+//             expect(response.statusCode).toBe(400)
+//         })
+//         // TODO: Add validation in API inputs
+//         // test("responds with 400 status", async () => {
+//         //     const response = await request(app)
+//         //         .post('/API/Elections')
+//         //         .set('Cookie', ['id_token=' + testInputs.user1token])
+//         //         .set('Accept', 'application/json')
+//         //         .send({ Election: testInputs.IncompleteElection, VoterIDList: [] })
+// 
+//         //     expect(response.statusCode).toBe(400)
+//         // })
+// 
+//     })
+//     describe("User is not logged in", () => {
+//         test("responds with 400 status", async () => {
+//             const response = await request(app)
+//                 .post('/API/Elections')
+//                 .set('Accept', 'application/json')
+//                 .send({ Election: testInputs.Election1, VoterIDList: [] })
+// 
+//             expect(response.statusCode).toBe(400)
+//         })
+// 
+//     })
+// })
+
+
+
+describe("Edit Election", () => {
+    // TODO: I'm making a lot of calls to add elections within the edit election tests
+    //       Ideally I'd like to setup the mocks without relying on the other apis, but I couldn't figure out a way to do that
```

Comment:
I couldn't figure out a good way around that either, but any set up API that we need should be well tested earlier in the test suit